scripts/funcotator/data_sources/createLiftoverForB37ToHg38.sh

```diff
@@ -0,0 +1,567 @@
+#!/usr/bin/env bash
+
+################################################################################
```

Comment:
It looks like you posted the result of this script.  Is that correct?

Reply:
Yes - the result is now in the repository as well.  I was on the fence about posting it, but I think having a cached file is a good idea.